lib/rbs/errors.rb

```diff
@@ -350,4 +350,42 @@ def location
       original.location
     end
   end
+
+  class MixinClassError < DefinitionError
+    attr_reader :type_name
+    attr_reader :member
+
+    def initialize(member:)
+      @type_name = member.name
```

Comment:
Should add a parameter `typename` which contains the name of the class/module definition.